internal/services/bot/bot_channel_directline_resource.go

```diff
@@ -64,12 +64,34 @@ func resourceBotChannelDirectline() *pluginsdk.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 
+						"application_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+
+						"block_user_upload_enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  false,
```

Comment:
Is there a specific reason to explicitly define the `Default` here, instead of just leaving it undefined as the other newly added boolean properties?

Reply:
Because API response explicitly returns default value "false" for this property.

Reply:
Since you are always setting these booleans in the request, if they are not specified in the configuration, then in your request, they'll be all `false` (no matter the `Default` is defined or not)?

Reply:
Yes. I removed "Default: false".